### DISCUSSION TITLE
代码尝试使用 inspect.getargspec，但是在 Python 3.10中，inspect 模块已经没有了 getargspec 属性，因此导致了 AttributeError，请问如何解决？

### DISCUSSION THREAD

**Discussion by KogWin** [open-compass/opencompass#1169] [Q&A]
Collecting prettytable (from opencompass==0.2.4)
  Downloading https://mirrors.aliyun.com/pypi/packages/3d/c4/a32f4bf44faf95accbbd5d7864ddef9e289749a8efbc3adaad4a4671779a/prettytable-3.10.0-py3-none-any.whl (28 kB)
Collecting pyext (from opencompass==0.2.4)
          oargspec = inspect.getargspec
                     ^^^^^^^^^^^^^^^^^^
      AttributeError: module 'inspect' has no attribute 'getargspec'. Did you mean: 'getargs'?
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.

**Comment by rainrambler** [comment]
看下这个帖子： https://github.com/pyinvoke/invoke/issues/833
